1-exercises/J-string-substring/exercise2.js

```diff
@@ -14,11 +14,11 @@ let names = [
   "Arron Graham",
 ];
 
-names[0] = names[0].substring();
-names[1] = names[1].substring();
-names[2] = names[2].substring();
-names[3] = names[3].substring();
-names[4] = names[4].substring();
+names[0] = names[0].substring(0, "Tamazin".length);
+names[1] = names[1].substring(0, "Jessica".length);
+names[2] = names[2].substring(0, "Tony".length);
+names[3] = names[3].substring(0, "Shae".length);
+names[4] = names[4].substring(0, "Arron".length);
 
 names.forEach((name) => {
```

Comment:
Nicely done! Please try to split the string in the for-loop based on the " " between name and surname :) It would be good practice for you.